src/core/render/compiler/image.js

```diff
@@ -0,0 +1,43 @@
+import { getAndRemoveConfig } from '../compiler'
+import { isAbsolutePath, getPath, getParentPath } from '../../router/util'
+
+export const imageCompiler = ({ renderer, contentBase, router }) => renderer.image = (href, title, text) => {
+  let url = href
+  let attrs = []
+
+  const { str, config } = getAndRemoveConfig(title)
+  title = str
+
+  if (config['no-zoom']) {
+    attrs.push('data-no-zoom')
+  }
+
+  if (title) {
+    attrs.push(`title="${title}"`)
+  }
+
+  const size = config.size
```

Comment:
```suggestion
  const { size } = config
```

Reply:
Instead of doing this, the suggestion from @anikethsaha has been applied